app/src/main/java/com/yml/womensafety/Alert.kt

```diff
@@ -0,0 +1,30 @@
+package com.yml.womensafety
+
+import android.content.Context
+import androidx.appcompat.app.AlertDialog
+
+/**
+ * This is used to display an Alert Dialog
+ * @param context context of the activity class
+ * @param title the title of alert dialog
+ * @param message the message to be displayed in alert dialog
+ * @param positiveMessage the message of the positive button in alert dialog
+ */
+object Alert {
+    fun alert(
+        context: Context,
+        title: CharSequence,
+        message: CharSequence,
+        positiveMessage: CharSequence
```

Comment:
Why you need class for this? You can just use util function or extension function here.